server/src/test/java/org/elasticsearch/health/node/HealthInfoTests.java

```diff
@@ -54,7 +60,7 @@ public static HealthInfo mutateHealthInfo(HealthInfo originalHealthInfo) {
                 HealthInfoTests::randomRepoHealthInfo
             );
         }
-        return new HealthInfo(diskHealth, dslHealth, repoHealth);
+        return new HealthInfo(diskHealth, dslHealth, repoHealth, FileSettingsHealthInfo.INDETERMINATE);
```

Comment:
I think it would be better to get some randomization in this class instead of using `INDETERMINATE` to have better coverage on the serialization of `FileSettingsHealthInfo`.